test/cases/api.sh

```diff
@@ -198,7 +198,10 @@ do
   if [[ "$COMPOSE_STATUS" != "pending" && "$COMPOSE_STATUS" != "running" ]]; then
     test "$COMPOSE_STATUS" = "success"
     test "$UPLOAD_STATUS" = "success"
-    test "$UPLOAD_TYPE" = "aws"
+    if [[ "$UPLOAD_TYPE" == "aws" ]]; then
+      AMI_ID=$(echo "$OUTPUT" | jq -r '.image_status.upload_status.target_results[0].options.ami' | cut -c1-4)
```

Comment:
I think the `target_results[0]` needs to be dropped.

Reply:
you're absolutely right, thank you!